src/request/components/request-detail-panel-execution.jsx

```diff
@@ -102,78 +137,129 @@ var MongoInsertCommand = React.createClass({
-            <div>Mongo DB URI: {mongoOperation.payload.connectionHost}:{mongoOperation.payload.connectionPort}/{mongoOperation.payload.database}/{mongoOperation.payload.collection} </div>
-            <div>Matched count: {mongoOperation.payload.matchedCount}</div>
-            <div>Modified count: {mongoOperation.payload.modifiedCount}</div>
-            <div>Upserted count: {mongoOperation.payload.upsertedCount}</div>
-            <div>Query: {JSON.stringify(mongoOperation.payload.query)}</div>
-            <div>Options: {JSON.stringify(mongoOperation.payload.options)}</div>
-        </div>;
+            <div className="tab-section-details-item">
+                <DetailListItem title="Operation" value={mongoOperation.payload.operation} />
+                <DetailListItem title="Mongo DB URI" value={this._makeUri(mongoOperation)}  />
+                <DetailListItem title="Matched count" value={mongoOperation.payload.matchedCount} />
+                <DetailListItem title="Modified count" value={mongoOperation.payload.modifiedCount} />
+                <DetailListItem title="Upserted count" value={mongoOperation.payload.upsertedCount} />
+                <DetailListItem title="Query" value={JSON.stringify(mongoOperation.payload.query)} />
+                <DetailListItem title="Options" value={JSON.stringify(mongoOperation.payload.options)} />
+            </div>;
         return details;
     }
 });
  
 var MongoDeleteCommand = React.createClass({
     mixins: [MongoCommandMixin],
-    
     getTitle: function() {
       return 'Mongo DB - Deleted ' + this.props.mongoOperation.payload.count +  ' documents';
     },
-    
     getDetails: function() {
         var mongoOperation = this.props.mongoOperation;
         var details =
-        <div> 
-            <div>Operation: {mongoOperation.payload.operation}</div>
-            <div>Mongo DB URI: {mongoOperation.payload.connectionHost}:{mongoOperation.payload.connectionPort}/{mongoOperation.payload.database}/{mongoOperation.payload.collection} </div>
-            <div>Query: {JSON.stringify(mongoOperation.payload.query)}</div>
-            <div>Options: {JSON.stringify(mongoOperation.payload.options)}</div>
-        </div>;
+            <div className="tab-section-details-item"> 
+                <DetailListItem title="Operation" value={mongoOperation.payload.operation} />
+                <DetailListItem title="Mongo DB URI" value={this._makeUri(mongoOperation)} />
+                <DetailListItem title="Query" value={JSON.stringify(mongoOperation.payload.query)} />
+                <DetailListItem title="Options" value={JSON.stringify(mongoOperation.payload.options)} />
+            </div>;
         return details;
     }
 });
 
+var HttpRequestItemHeaderList = React.createClass({ 
+    render: function() {
+        var headers = this.props.headers;
+        if (headers) {
+            return (
+                    <div>{_.map(headers, function(value, key) {
```

Comment:
Just out of curiosity, is there a reason we use `_.map(headers` instead of `headers.map(`?